version.properties

```diff
@@ -1,5 +1,5 @@
 #Fri May 27 05:08:14 UTC 2022
-SDK_VERSION=3.2.1
+SDK_VERSION=3.2.2
 VERSION_MAJOR=3
 VERSION_MINOR=2
-VERSION_PATCH=1
+VERSION_PATCH=2
```

Comment:
Do we want to include this right now or should we wait till we cut a release from this branch?

Reply:
this PR merge to the release branch will trigger the release. we won’t be manually running the update-version workflow